src/main/java/nextstep/courses/domain/CoverImageFileName.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.InvalidImageNameException;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CoverImageFileName {
+
+    private static final Pattern EXTENSION_PATTERN = Pattern.compile(".*\\.(jpeg|jpg|png|gif)$");
```

Comment:
```suggestion
    private static final Pattern EXTENSION_PATTERN = Pattern.compile(".*\\.(jpeg|jpg|png|gif)$");
```

지금 구조도 큰 문제가 안되지만,
`Enum`을 활용해서 Pattern 객체를 만드는것도 가능하겠네요~(반영을 위한 코멘트는 아닙니다)

Reply:
좋은거 같습니다 ㅎㅎ https://github.com/next-step/java-lms/pull/238/commits/828c4cc16fd17a82882c3df76ccabd8be6ea37fc 해당 커밋에서 Enum으로 관리할 수 있도록 수정했습니다!  또한 https://github.com/next-step/java-lms/pull/238/commits/77a388d57f5821a525156b8ff5c68b135145879b 해당 enum에 대한 테스트 코드도 추가했습니다 :)